KomaMRIFiles/src/Phantom/Phantom.jl

```diff
@@ -5,108 +5,108 @@ Reads a (.phantom) file and creates a Phantom structure from it
 """
 function read_phantom(filename::String)
     fid = HDF5.h5open(filename, "r")
-    phantom_fields = []
-    version = read_attribute(fid, "Version")
     dims = read_attribute(fid, "Dims")
     Ns = read_attribute(fid, "Ns")
+    # Version
+    file_version = VersionNumber(read_attribute(fid, "Version"))
+    program_version = pkgversion(KomaMRIFiles)
+    if file_version.major != program_version.major
+        @warn "Version mismatch detected: $file_version (file) vs $program_version (Koma)
+         This may lead to compatibility issues. Please update the file or the program."
+    end
```

Comment:
```suggestion
    if file_version.major != program_version.major
        @warn "KomaMRIFiles: Version mismatch detected: $file_version (used to write .phantom) vs $program_version (installed)
         This may lead to compatibility issues. "
    end
```